updating/updating-restricted-network-cluster/restricted-network-update-osus.adoc

```diff
@@ -98,20 +98,22 @@ include::modules/update-service-configure-cvo.adoc[leveloffset=+3]
 See xref:../../networking/enable-cluster-wide-proxy.adoc#nw-proxy-configure-object[Enabling the cluster-wide proxy] to configure the CA to trust the update server.
 ====
 
-[id="update-service-delete-service"]
-== Deleting an OpenShift Update Service application
+[id="nexts-steps_updating-restricted-network-cluster-osus"]
+== Next Steps
 
-You can delete an OpenShift Update Service application by using the {product-title} web console or CLI.
+Before updating your cluster, you should have performed the following tasks:
 
-include::modules/update-service-delete-service-web-console.adoc[leveloffset=+2]
+* You have configured the Cluster Version Operator (CVO) to use your locally installed OpenShift Update Service application.
+* You have applied the release image signature config map for the new release to your cluster.
+* You have mirrored the target release images onto your local registry.
```

Comment:
We might need to mention that you have mirrored the current release image that your cluster is running on onto your local registry.

Reply:
This is right. I suggest mirroring the current release and the future release the cluster will upgrade to.

Reply:
Sounds good, I added a mention of the current release images to this bullet point. @shellyyang1989 let me know what you think!